helm/scylla-operator/values.yaml

```
@@ -42,6 +42,11 @@ webhook:
   # If not set and createSelfSignedCertificate is true, a name is generated using fullname
   certificateSecretName: ""
 
+  resources:
+    requests:
+      cpu: 100m
```

```suggestion
      cpu: 10m
```
Lets leave existing default value.